scripts/semconv/generate.sh

```diff
@@ -28,7 +32,8 @@ docker run --rm \
   code \
   --template /templates/SemanticAttributes.ts.j2 \
   --output /output/SemanticAttributes.ts \
-  -Dclass=SemanticAttributes
+  -Dclass=SemanticAttributes \
+  -Dcls_prefix=SEMATTRS
```

Comment:
Passing the "prefix" for the constants here but not the individual "constant" strings as (at least on windows with git bash shell) the command line gets exceeded and the tool (via docker) fails to run 😢